### DISCUSSION TITLE
ISPRS Potsdam for Prediction

### DISCUSSION THREAD

**Discussion by Pelope9832** [azavea/raster-vision#2242] [Q&A]
Hi! I discovered rastervision a couple of days ago, and I'm trying to do a prediction as the tutorial https://docs.rastervision.io/en/stable/usage/tutorials/pred_and_eval_ss.html with the ISPRS Potsdam model, because I want to segment trees and low vegetation, but I don't know how change the tutorial. Could you help me, please?

**Comment by AdeelH** [comment]
You might find this tutorial notebook helpful: https://github.com/Element84/ml-workshop/blob/external-workshop/notebooks/01-prediction_with_existing_models/01-basic_segmentation.ipynb. It uses a model trained on ISPRS Potsdam.

**Reply by Pelope9832** [reply]
Thank you very much
